### PR TITLE
Color the coordinates transform graph edges by transform type

### PR DESCRIPTION
**Pull request by adrn**
This colors the edges in the `graphviz`-generated visualization of the frame transform graph based on the type of transformation. I'm not sure this needs to be in the documentation, but @eteq and I needed to look at this for thinking about velocity support so I implemented it anyways. Attached is an image of the colored graph. The colors were picked using [colorbrewer2](http://colorbrewer2.org/#type=qualitative&scheme=Dark2&n=3), 3 qualitative colors, colorblind-safe.

EDIT: (I only have 5 seconds now but here's an update of what it looks like, without the priorities. sorry for the volume showing)

screen shot 2017-05-10 at 16 50 23

cc @eteq

### CODE REVIEW

**Comment by adrn** [comment]
BTW: I couldn't figure out how to make a good legend using graphviz. If anyone has ideas about that, let me know.

**Comment by pllim** [comment]
Do you also plan to address astropy/astropy-helpers#276 or is that out of scope?

**Comment by adrn** [comment]
@pllim I don't, because this stretched my knowledge of `graphviz` as it is!

**Comment by astrofrog** [comment]
Nice!

**Comment by eteq** [comment]
@adrn - here's an example of a fancy legend in graphviz: https://www.flickr.com/photos/kentbye/1155558045 ... maybe that can be adapted?

**Comment by adrn** [comment]
@eteq that example must use some other custom code that I can't find (`InteractionColorCorrelatedCoexpression`). When I generate the same graph locally, I get: 
test

(i.e. no reasonable legend)

**Comment by eteq** [comment]
Ugh, ok. Not worth working too hard on this right now.  I'd say just drop the legend in the graph.  But maybe you can insert the text with the colors in the docstring?

**Comment by adrn** [comment]
Here's how it looks now, with an HTML legend. Included another volume box for posterity:
image

**Comment by adrn** [comment]
Now with arrows...
image

**Comment by adrn** [comment]
@bsipocz poke @eteq until he merges this - I refuse to change the arrow styles any more.

**Comment by eteq** [comment]
Fortunately... Those are good!

**Comment by eteq** [comment]
Also, thanks for including the volume on the last one, @adrn.  Definitely a critical touch.

**Comment by saimn** [comment]
It seems this one broke the python 2 build: https://travis-ci.org/astropy/astropy/jobs/231394112

**Comment by bsipocz** [comment]
I've rerunning travis on the merge commit now, it was skipped yesterday night when this was merged: https://travis-ci.org/astropy/astropy/builds/231371019

File: astropy/coordinates/builtin_frames/__init__.py [hunk 1]
```diff
@@ -79,16 +83,40 @@ def _make_transform_graph_docs():
...
+
+    # colors are in dictionary at the bottom of transformations.py
+    from ..transformations import trans_to_color
+    html_list_items = []
+    for cls,color in trans_to_color.items():
+        block = """
+            <li style='list-style: none;'>
+                <p style="font-size: 12px;line-height: 24px;font-weight: normal;color: #848484;padding: 0;margin: 0;">
+                    <b>{0}:</b>
+                    <span style="font-size: 24px; color: {1};"><b>➝</b></span>
```
Comment (saimn) [inline astropy/coordinates/builtin_frames/__init__.py]: I guess the unicode arrow is the issue (for Python 2). This should a unicode string.

**Pull request merged**